internal/run/source.go

```diff
@@ -51,6 +51,7 @@ type SourceResult struct {
 	LintResult   *validation.ValidationResult
 	ChangeReport *reports.ReportResult
 	Diagnosis    *suggestions.Diagnosis
+	OutputPath   string
```

Comment:
Why does this need to be part of source result. Is this the output path of the source. Maybe a comment clarifying